src/dom.rs

```diff
@@ -606,6 +608,33 @@ impl<A> DomBuilder<A> where A: AsRef<Node> {
     }
 }
 
+impl<A> DomBuilder<A> where A: AsRef<Node> {
+    #[inline]
+    pub fn child_signal<B>(mut self, signal: B) -> Self
```

Comment:
The implementation is mostly based on `children_signal_vec()`/`operations::insert_children_signal_vec()`.